### DISCUSSION TITLE
How to push "Created By" to webhook?

### DISCUSSION THREAD

**Discussion by burnoutberni** [Budibase/budibase#2366] [Help]
Whenever a new row is created in a Budibase table, I push the whole row to an API via the webhooks feature:

```
{
  "posting": {{ JSONstringify trigger.row }},
  …
}
```

Unfortunately the resulting JSON string does not contain the `Created By` field, but I also need this information. Is there any way to do this?

**Comment by mjashanks** [comment]
Hey Bernhard. Unfortunately, it's not possible. 

I'll need to log an issue. Just so I understand, what data do you need from the "Created By" field? `email`, `name`?

**Reply by burnoutberni** [reply]
@mjashanks Thanks for the response. So, generally I was assuming I'd get the same row data as if I was doing an export. The export also contains metadata such as:

```
    …
    "Created At": "2021-07-19T16:13:12.259Z",
    "Updated At": "2021-07-19T16:13:12.259Z",
    "type": "row",
    "Created By": [
      {
        "_id": "ro_ta_users_us_bddb708bf0ce4e7b91a175feddbc28b5",
        "primaryDisplay": "hello@nini.su"
      }
    ],
    "Updated By": [
      {
        "_id": "ro_ta_users_us_bddb708bf0ce4e7b91a175feddbc28b5",
        "primaryDisplay": "hello@nini.su"
      }
    ]
```

Some of this is also present in the webhook row, but some of it – such as "Created By" – is missing. But yeah, adding the name as well would be good, probably also for the export.
